### PR TITLE
Add Bio-Virologia

### PR DESCRIPTION
**Pull request by AimerGDiaz**
Creating an access port to BioViral bookdown, currently here 
(https://aimergdiaz.github.io/Bio-Virologia/)

### CODE REVIEW

**Comment by AimerGDiaz** [comment]
Thanks in advance

File: R/staging.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+http://biovirologia/
```
Comment (cderv) [inline R/staging.txt:1]: Seems like the URL should be https://aimergdiaz.github.io/Bio-Virologia/, right ?

**Review by cderv — CHANGES_REQUESTED** [review]
Thanks for adding your book. Quick question on the URL

**Review by AimerGDiaz — COMMENTED** [review]
Corrected , thanks

**Review by cderv — APPROVED** [review]
Thanks I'll merge and then this should be live on the website next thursday on the weekfly update.

**Pull request merged**